CI/appveyor.after_build.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/sh
+
+# This assembles all the files we need into a package sub-directory
+
+echo "Running appveyor.after_build.sh shell script..."
+echo ""
+
+if [ ${APPVEYOR_REPO_NAME} != "Mudlet/Mudlet" ]; then
+    # Only run this code on the main Mudlet Github repository - do nothing otherwise:
+    echo "This does not appear to be running on the main Mudlet repository, packaging is not appropriate....!"
+    echo ""
+    exit 0
+fi
+
+# Source/setup some variables (including PATH):
+. $(/usr/bin/cygpath --unix ${APPVEYOR_BUILD_FOLDER}/CI/appveyor.set-build-info.sh)
+
+echo "Moving to packaging directory: $(/usr/bin/cygpath --windows ${APPVEYOR_BUILD_FOLDER}/package)"
+cd $(/usr/bin/cygpath --unix ${APPVEYOR_BUILD_FOLDER}/package)
+echo "  it contains:"
+ls -l
+echo ""
+
+# Since Qt 5.14 using the --release switch is broken (it now seems to be
+# assumed), --debug still seems to work.
+# https://bugreports.qt.io/browse/QTBUG-80806 seems relevant.
+echo "Running windeployqt..."
+${MINGW_INTERNAL_BASE_DIR}/bin/windeployqt --no-virtualkeyboard mudlet.exe
+echo ""
+
+# To determine which system libraries have to be copied in it requires
+# continually trying to run the executable on the target type system
+# and adding in the libraries to the same directory and repeating that
+# until the executable actually starts to run. Alternatively running
+# ldd ./mudlet.exe | grep "/mingw32" {for the 32 bit case, use "64" for
+# the other one} inside an Mingw32 (or 64) shell as appropriate will
+# produce the libraries that are likely to be needed below. Unfortunetly
+# this process is a little recursive in that you may have to repeat the
+# process for individual librarys. For ones used by lua modules this
+# can manifest as being unable to "require" the library within lua
+# and doing the above "ldd" check revealed that "zip.dll" needed
+# "libzzip-0-13.dll" and "luasql/sqlite3.dll" needed "libsqlite3-0.dll"!
```

Comment:
I think you will need to run the `ntldd` thing on the mentioned lua libraries as well.

Reply:
Trouble was I didn't spot that I had to install the 32/64 bit package that provides it...

Reply:
🤔 Checking the listed ones reveals the (handled) non-native lib requirements of:
* `lfs.dll`:
  * `lua51.dll`
* `lpeg.dll`
  * `lua51.dll`
* lsqlite3.dll
  * `lua51.dll`
  * `libsqlite3-0.dll`
* lua-utf8.dll
  * `lua51.dll`
* rex_pcre.dll
  * `lua51.dll`
  * `libpcre-1.dll`
* `yajl.dll`
  * `lua51.dll`
  * `libyajl.dll`
* `luasql/sqlite3.dll`
  * `lua51.dll`
  * `libsqlite3-0.dll`
* `brimworks/zip.dll` (changed to that one from the Kepler project's one in one of the last few commits)
  * `lua51.dll`
  * `libzip.dll`

I have run through the other libraries that are currently listed explicitly for various reasons and we seem to be covered. 🤞

Reply:
> brimworks/zip.dll (changed to that one from the Kepler project's one in one of the last few commits)

Why? Keep to the scope, please :(

Reply:
Agree. This is scope leak.

Reply:
:mag_right: :thinking: The other CIs are split over `lua-zip` v.s. `luazip`:
* MacOs - changed to it in https://github.com/Mudlet/installers/commit/fedaa2e8cd3476b573db2a46b2e75772eb03c9d8 18 Apr 2020 IIRC it was because the `luazip` version has a dependency on `zzliblip`/`libzzip`/`zzip` (the name seems to vary in different places but it does always have two "zz"s) that became problematic to compile on that platform at least.
* Linux  - refers to the library directly in the `.travis.yml` - confusingly the package is called `lua-zip` but it is the Kepler projects one (luarock `luazip`) as opposed to the Brimworks one (luarock `lua-zip`) - furthermore the *Debian* package concerned is based upon the version **1.2.3** code that was extracted from an SVN repository and place at https://github.com/luaforge/luazip where it has remained untouched since 2007. That code has been forked and revised at https://github.com/mpeterv/luazip - most recently in 2017 to a version **1.2.7** which is what the `luazip` rock is based upon. {So the `luazip` rock is based on code that is ten years newer than the Debian package!} in comparison the `lua-zip` rock is at a version **0.2.0** and is unrelated to the original Kepler project code - but it was last revised in 2019...

Given that I am constructing a new build system (and not compiling the luazip code from source as the prior Windows CI process did as it downloads and compiles https://github.com/rjpcomputing/luazip/archive/master.zip which is actually redirected to https://github.com/mpeterv/luazip {i.e. the source of the `luazip` rock NOT the older form that the Linux build uses}) I have to say, this ain't scope leak, but a reasonable choice out of a couple of alternatives **both of which our application is already built to handle**.

Reply:
It's changing more things than are needed - Windows was built with the other one so it would be safer to stick with it. But let's try it and see if it works well.